packages/server/src/adapters/node-http/content-type/json/getPostBody.ts

```diff
@@ -9,6 +9,21 @@ export async function getPostBody(opts: {
 }): Promise<BodyResult> {
   const { req, maxBodySize = Infinity } = opts;
   return new Promise((resolve) => {
+    if (
+      !req.headers['content-type']?.startsWith('application/json') &&
+      req.method !== 'GET' &&
+      req.method !== 'OPTIONS' &&
+      req.method !== 'HEAD'
```

Comment:
DELETE cannot have body either right?

Reply:
no, but DELETE should definitely be CSRF-prevented too as it's very much a mutation